doc/source/workflows/basics.rst

```diff
@@ -203,12 +211,12 @@ For example,
 
 By default, each task will be given a name generated by the library, ``<MODULE_NAME>.<FUNC_NAME>``. In the example above, ``step`` is given as the task name for function ``simple``.
 
-When the task name duplicates, we append ``_n`` to the name by the order of execution as it task ID. So the initial task get the ID ``step``, the second one get ``step_1``, and this goes on for all later tasks.
+When the task name duplicates, we append ``_n`` to the name by the order of execution as its task ID. So the initial task gets the ID ``step``, the second one gets ``step_1``, and this goes on for all later tasks.
 
 Error handling
 --------------
 
-Workflows provides two ways to handle application-level exceptions: (1) automatic retry (as in normal Ray tasks), and (2) the ability to catch and handle exceptions.
+Workflow provides two ways to handle application-level exceptions: (1) automatic retry (as in normal Ray tasks), and (2) the ability to catch and handle exceptions.
 
 - If ``max_retries`` is given, the task will be retried for the given number of times if the workflow task failed.
```

Comment:
Is max_retries recursive? For example, if function f calls function g, and max retries for f is 5 and that for g is 2, then is it possible for g to be called twice for each invocation of f - meaning 10 times overall?

Reply:
I also feel it's a little bit confusion here. In the case of f call g, it's actually a dynamic workflow here the workflow is generated on the runtime. I think it won't be recursive. @suquark to confirm.